runtimes/pythonrt/py.go

```diff
@@ -143,66 +143,88 @@ func pyExeInfo(ctx context.Context, exePath string) (exeInfo, error) {
 }
 
 type pyRunInfo struct {
-	rootDir  string
-	sockPath string
-	lis      net.Listener
-	proc     *os.Process
+	userRootDir string
+	pyRootDir   string
+	sockPath    string
+	lis         net.Listener
+	proc        *os.Process
+}
+
+func (ri *pyRunInfo) Cleanup() {
+	if ri.userRootDir != "" {
+		os.RemoveAll(ri.userRootDir)
+	}
+
+	if ri.pyRootDir != "" {
+		os.RemoveAll(ri.pyRootDir)
+	}
 }
 
 type runOptions struct {
 	log            *zap.Logger
-	pyExe          string
-	tarData        []byte
-	rootPath       string
-	env            map[string]string
+	pyExe          string            // Python executable to use
+	tarData        []byte            // tar data from build stage
+	rootPath       string            // simple.py:greet
```

Comment:
`rootPath` is a confusing name. It's associated with directories rather than filenames and function names. It's not a showstopper, but what about `entryPoint` or `call`?

Reply:
In the `Call` function signature this is called `mainPath`. Changed to `entryPoint`